utility/MathHelper.cpp

```diff
@@ -148,6 +148,19 @@ void Convert::VertexToRecast(const Vector3& input, float* output)
     output[2] = -x;
 }
 
+void Convert::RecastToVertex(const float* const input, Vector3& output)
```

Comment:
How is this different than `VertexToWow`?

Reply:
It isn't, I missed that `VertexToWow` existed.